### PR TITLE
Add Ingólfur Template

### PR DESCRIPTION
**Pull request by fidellim**
Technologies used:

- Vuejs
- Vanilla CSS
- jQuery
- Animation on Scroll library
- Parallax.js

<details><summary>Click to view new template in root page</summary>
<p>

https://user-images.githubusercontent.com/26820513/229114805-03aa2bbb-136c-4804-9b61-37d3c3cb8bf0.mp4

</p>
</details> 

<details><summary>Click to view new template in README</summary>
<p>

https://user-images.githubusercontent.com/26820513/229114845-efb0f36b-6310-4355-b711-80644bd19b1a.mp4

</p>
</details> 

<details><summary>Click to view OG tags</summary>
<p>

</p>
</details> 

<details><summary>Click to view template</summary>
<p>

### Desktop View
https://user-images.githubusercontent.com/26820513/229116628-517e0f33-aaa4-4cf6-8596-248e8d5a47d1.mp4

### Mobile View
https://user-images.githubusercontent.com/26820513/229116685-32ab53c2-e7a6-4673-8b83-18a56de60040.mp4

</p>
</details>

### CODE REVIEW

**Comment by jofftiquez** [comment]
Hi @fidellim thank you for this again. I noticed you are using bandcam, have you checked loom? It's a great alternative

File: ingolfur/index.html [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+
+        <div class="row hero-content">
+
+            <div class="column large-full">
+
+                <h1>
+                Hi there, I'm Jane Smith, <br>
+                a creative web designer <br>
+                and frontend developer <br>
+                located in Philippines.
```
Comment (jofftiquez) [inline ingolfur/index.html:86]: Replace "located" with either "based in the Philippines" or "form the Philippines" Thanks.

**Review by jofftiquez — CHANGES_REQUESTED** [review]
Just one small change for the hero panel text

**Comment by fidellim** [comment]
> Hi @fidellim thank you for this again. I noticed you are using bandcam, have you checked loom? It's a great alternative

Thank you sir @jofftiquez for the recommendation. Will have a look at it!

File: ingolfur/index.html (hunk 1 above)
Comment (fidellim) [reply ingolfur/index.html]: Hello sir @jofftiquez, I have updated the text in the hero section. Please refer to the image below. Thank you!

image

**Review by jofftiquez — APPROVED** [review]

**Pull request merged**